test/kata7.getEmployerRole.test.js

```diff
@@ -1,5 +1,26 @@
 const { getEmployerRole } = require("../src");
 
 describe("getEmployerRole", () => {
-  test("returns the employee's role in the company", () => {});
+  it("returns the employee's role in the company", () => {
+    const employees = [
+      {
+        name: "Satti",
+
+        role: "Developer",
+      },
+      {
+        name: "Jenny",
+
+        role: "Sales Associate",
+      },
+      {
+        name: "Javid",
+
+        role: "Human Recommended Reading Assistant",
+      },
+    ];
+    expect(getEmployerRole("Javid", employees)).toEqual(
+      "Human Recommended Reading Assistant"
+    );
+  });
```

Comment:
Small formatting change here, you dont need the clear lines inside the objects, and its good to leave a clear line between the setup part of the tests and the start of the assertions:
```
  it("returns the employee's role in the company", () => {
    const employees = [
      {
        name: "Satti",
        role: "Developer",
      },
      {
        name: "Jenny",
        role: "Sales Associate",
      },
      {
        name: "Javid",
        role: "Human Recommended Reading Assistant",
      },
    ];

    expect(getEmployerRole("Javid", employees)).toEqual(
      "Human Recommended Reading Assistant"
    );
  });
```